roc/receiver.go

```diff
@@ -171,7 +171,12 @@ type Receiver struct {
 
 // Open a new receiver.
 // Allocates and initializes a new receiver, and attaches it to the context.
-func OpenReceiver(context *Context, config ReceiverConfig) (*Receiver, error) {
+func OpenReceiver(context *Context, config ReceiverConfig) (receiver *Receiver, err error) {
+	logWrite(LogDebug, "entering OpenReceiver(): config=%+v context=%p", config, context)
```

Comment:
nit: everywhere we log context, let's make it the first in the log message, for consistency.